sopel/loader.py

```diff
@@ -158,7 +158,9 @@ def clean_callable(func, config):
     func.unblockable = getattr(func, 'unblockable', False)
     func.priority = getattr(func, 'priority', 'medium')
     func.thread = getattr(func, 'thread', True)
-    func.rate = getattr(func, 'rate', 0)
```

Comment:
Remove `rate` will break backwards compatibility with jenni modules. Maybe alias `rate` to `user_rate` in the getattr?